docs/en/operations/settings/settings.md

```diff
@@ -404,6 +404,38 @@ Possible values:
 
 Default value: 0.
 
+## partial_merge_join_optimizations {#partial_merge_join_optimizations}
+
+Enables of disables optimizations in partial merge join algorithm for [JOIN](../../sql-reference/statements/select/join.md) queries.
+
+Optimizations:
+
+- Skipping of  unneeded right blocks instead of pass through them.
```

Comment:
```suggestion
- Skipping of unneeded right blocks instead of going through them.
```

Reply:
I'd prefer not to describe that sort of optimisations we really have. It depends on version.